cpuid/src/cpu_leaf.rs

```diff
@@ -124,42 +124,70 @@ pub mod leaf_0x7 {
             // Intel® Resource Director Technology (Intel® RDT) Monitoring
             pub const RDT_M_SHIFT: u32 = 12;
             // 13 = Deprecates FPU CS and FPU DS values if 1
-            // 14 = MPX (Intel® Memory Protection Extensions)
+            // Memory Protection Extensions
+            pub const MPX_SHIFT: u32 = 14;
             // RDT = Intel® Resource Director Technology
             pub const RDT_A_SHIFT: u32 = 15;
             // AVX-512 Foundation instructions
             pub const AVX512F_SHIFT: u32 = 16;
+            // AVX-512 Doubleword and Quadword Instructions
+            pub const AVX512DQ_SHIFT: u32 = 17;
             pub const RDSEED_SHIFT: u32 = 18;
             pub const ADX_SHIFT: u32 = 19;
             // 20 = SMAP (Supervisor-Mode Access Prevention)
-            // 21 & 22 reserved
-            // 23 = CLFLUSH_OPT (flushing multiple cache lines in parallel within a single logical processor)
-            // 24 = CLWB (Cache Line Write Back)
+            // AVX512IFMA = AVX-512 Integer Fused Multiply-Add Instructions
+            pub const AVX512IFMA_SHIFT: u32 = 21;
+            // 21 = PCOMMIT intruction
+            // 22 reserved
+            // CLFLUSHOPT (flushing multiple cache lines in parallel within a single logical processor)
+            pub const CLFLUSHOPT_SHIFT: u32 = 23;
+            // CLWB = Cache Line Write Back
+            pub const CLWB_SHIFT: u32 = 24;
             // PT = Intel Processor Trace
             pub const PT_SHIFT: u32 = 25;
-            // AVX512CD = AVX512 Conflict Detection
+            // AVX512PF = AVX512 Prefetch Instructions
+            pub const AVX512PF_SHIFT: u32 = 26;
+            // AVX512ER = AVX-512 Exponential and Reciprocal Instructions
+            pub const AVX512ER_SHIFT: u32 = 27;
+            // AVX512CD = AVX-512 Conflict Detection Instructions
             pub const AVX512CD_SHIFT: u32 = 28;
             // Intel Secure Hash Algorithm Extensions
             pub const SHA_SHIFT: u32 = 29;
-            // 30 - 32 reserved
+            // AVX-512 Byte and Word Instructions
+            pub const AVX512BW_SHIFT: u32 = 30;
+            // AVX-512 Vector Length Extensions
+            pub const AVX512VL_SHIFT: u32 = 31;
         }
 
         pub mod ecx {
             // 0 = PREFETCHWT1 (move data closer to the processor in anticipation of future use)
-            // 1 = reserved
+            // AVX512_VBMI = AVX-512 Vector Byte Manipulation Instructions
+            pub const AVX512_VBMI: u32 = 1;
```

Comment:
Why is this const (and a few others) missing a suffix? It looks a bit messy for some bits to have a `_SHIFT`, others a `_BITINDEX`, and others no suffix.

Reply:
It's an honest mistake. All shitfs should have _SHIFT. For the ones that are bit ranges, they should have the _BITRANGE suffix.